pkg/cidata/cidata.TEMPLATE.d/lima.env

```diff
@@ -15,3 +15,6 @@ LIMA_CIDATA_CONTAINERD_SYSTEM=1
 LIMA_CIDATA_CONTAINERD_SYSTEM=
 {{- end}}
 LIMA_CIDATA_SLIRP_GATEWAY={{ .SlirpGateway }}
+{{- range $key, $val := .EnvironmentVariables}}
+{{$key}}={{$val}}
+{{- end}}
```

Comment:
The env vars here are only used for the init script of Lima.
To apply the env vars globally, `/etc/environment` need to be updated.

The file path may differ across guest distros, so we should have an integration tests. (`hack/test-example.sh`)